bulk/__main__.py

```diff
@@ -1,32 +1,34 @@
-import pathlib 
+import pathlib
 
-import typer 
-from tornado.ioloop import IOLoop
-from bokeh.util.browser import view
+import typer
 from bokeh.server.server import Server
+from bokeh.util.browser import view
+from tornado.ioloop import IOLoop
 
 from bulk.text import bulk_text
 
-
 app = typer.Typer(
     name="bulk",
     add_completion=False,
     help="Tools for bulk labelling.",
     no_args_is_help=True,
 )
 
+
 @app.command("version")
 def version():
-    print("0.1.0")
+    print("0.1.1")
```

Comment:
This version implementation is currently wrong, it needs to pick the actual version from the package. I'll fix that later, but it's preferable to leave this untouched. That might just prevent a few merge conflicts.

Reply:
Okay 😃